### ISSUE TITLE
history_graph: de-selecting sensor, re enables after auto-data-refresh

### ISSUE THREAD

**Issue by oz9els** [home-assistant/frontend#9755]
### The problem

Disabled graph, auto re-enables in History charts.

I have a chart with 3 graphs (lines/curves), one for each of 3 sensors.
![image](https://user-images.githubusercontent.com/61284428/125062871-259ee100-e0af-11eb-8a52-45d295aa6c11.png)

When deselecting a sensor to show in the graph, the graph disappears (as expected) - Now I have 2 graphs.
![image](https://user-images.githubusercontent.com/61284428/125063001-48c99080-e0af-11eb-845d-bfc5da3708de.png)

When values get updated, this also updates the graphs (as expected)
The problem is that when a value updates, the graphs updates, BUT the deselected value reappears and i now got 3 graphs again. With the deselected graph still in my chart, even thou it is striked-through
![image](https://user-images.githubusercontent.com/61284428/125063115-626ad800-e0af-11eb-990a-f1c7fd23204d.png)

I can see the graphs have had been updated, probably missed this issue ;-)
Thanks for your great effort..### 

### What is version of Home Assistant Core has the issue?

core-2021.07.1

### What was the last working version of Home Assistant Core?

core-2021.06.x

### What type of installation are you running?

Home Assistant OS

### Integration causing the issue

Lovelace (history_graph card)

### Link to integration documentation on our website

https://www.home-assistant.io/lovelace/history-graph/

### Example YAML snippet

```yaml
type: history-graph
entities:
  - entity: sensor.power_minus
    name: Uforbrugt
  - entity: sensor.power_solar
    name: Solpaneler
  - entity: sensor.power_plus
    name: import
refresh_interval: 0
hours_to_show: 1
```

### Anything in the logs that might be useful for us?

```txt
no logs relevant
```

**Comment by chairstacker** [comment]
Looks like it's been resolved in 2021.9.4 - maybe even sooner.

**Comment by oz9els** [comment]
Also looks fine here - THANKS guys..

**Comment by oz9els** [comment]
Solved in latest release 2021.09.4 THANKS
